data/src/main/java/com/example/data/database/MovieDao.kt

```diff
@@ -5,12 +5,19 @@ import androidx.room.Insert
 import androidx.room.OnConflictStrategy
 import androidx.room.Query
 import com.example.data.entity.MovieEntity
+import com.example.domain.util.TabsEnum
 
 @Dao
 interface MovieDao {
-    @Query("SELECT * FROM movie_table")
-    fun getNowPlayingMovies(): List<MovieEntity>
+    @Query("SELECT * FROM movie_table WHERE tab = :tabsEnum")
+    fun getNowPlayingMovies(tabsEnum: TabsEnum): List<MovieEntity>
 
     @Insert(onConflict = OnConflictStrategy.REPLACE)
     fun insertNowPlayingMovie(movieEntity: MovieEntity)
+
+    @Query("SELECT * FROM movie_table WHERE tab = :tabsEnum")
+    fun getTopRatedMovies(tabsEnum: TabsEnum): List<MovieEntity>
```

Comment:
this method is not necessary as you are doing the same in the lines 12 method

Reply:
it's true. fixed